pkg/metrics/predefined_metrics.go

```diff
@@ -0,0 +1,133 @@
+package metrics
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
```

Comment:
Nitpicking: may be rename the import flpapi to avoid confusion between operator api and flp api